test/framework/fixtures/libgoalFixture.go

```diff
@@ -499,3 +502,50 @@ func (f *LibGoalFixture) MinFeeAndBalance(round uint64) (minFee, minBalance uint
 	}
 	return params.MinTxnFee, minBalance, nil
 }
+
+// ProofRespToProof returns a proof for usage in merkle array verification from the inputted API proof query response.
```

Comment:
I'm now convinced that these functions shouldn't actually be here, as they should also be used in the unit tests (see the TODO in handlers_test). Where do you think they should go? @algoidan @Aharonee

Reply:
We've decided to move the path builder function to the merklearray